packages/mendel-development-middleware/package.json

```diff
@@ -14,9 +14,9 @@
   },
   "dependencies": {
     "path-to-regexp": "^1.7.0",
-    "mendel-config": "3.0.0",
+    "mendel-config": "3.0.1",
     "mendel-development": "^1.1.0",
-    "mendel-exec": "3.0.0",
-    "mendel-pipeline": "3.0.0"
+    "mendel-exec": "3.0.1",
+    "mendel-pipeline": "3.0.1"
```

Comment:
I believe it should be `^3.0.0` actually this way we don't need to bump for minor versions of everything. 

Have you tried not bumping and just installing prune and uglify directly? If it does not work, can we fix by making this definition loose?